rmw_connext_cpp/src/identifier.cpp

```diff
@@ -12,9 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "identifier.hpp"
+#include "rmw_connext_cpp/identifier.hpp"
 
-extern "C"
-{
```

Comment:
does it hurt to keep it ? otherwise if we ever add functions here we will have to remember to add back the extern